soccer/gameplay/plays/restarts/placement.py

```diff
@@ -0,0 +1,50 @@
+import standard_play
+import behavior
+import robocup
+import main
+import constants
+import tactics.line_up
+import tactics.our_placement
+
+
+# one robot places the ball, the others just line up and wait
+class Placement(standard_play.StandardPlay):
+    def __init__(self):
+        super().__init__(continuous=True)
+
+        self.remove_subbehavior('defense')
```

Comment:
Ahh ok, makes sense.  Maybe we should have this play subclass the normal `Play` class then.  The `StandardPlay` exists mostly to handle defense.